app/src/main/java/com/example/sabonit/CartActivity.java

```diff
@@ -96,4 +97,17 @@ public void onProductDeleteClick(View view, int position) {
         updateCartAdapter();
     }
 
+    public void goToOrder(View view) {
+        showHowItWorksDialog();
+//        Intent intent = new Intent(this, OrderActivity.class);
+//        startActivity(intent);
```

Comment:
You had to write somewhere that the work is in progress, but nevermind...
At least you wrote it in a modular way

Reply:
Yes, I wrote WIP in the commit message